### PR TITLE
680 - entrypoint support

### LINKED ISSUE #680: The babylon-enabled micheline parser throws away entrypoint info for "parameters"
related to #618 

After the change we can correctly parse the post-babylon format for transactions' `parameters`.
During such process though, we're simply discarding information regarding the `entrypoint` of the contract, which is critical to users.

We need to restore such information, evaluating if it can be correctly parsed as part of the whole micheline expression, or if it's to be exposed as part of another wrapping data structure that shows entrypoint and value as separate entries

### Example case
code originally unparsed which showed as
```json
{
"parameters": {
    "entrypoint": "transfer",
    "value": {
      "prim": "Pair",
      "args": [
        {
          "string": "tz1d75oB6T4zUMexzkr5WscGktZ1Nss1JrT7"
        },
       {
          "prim": "Pair",
          "args": [
            {
              "string": "tz1Mj7RzPmMAqDUNFBn5t5VbXmWW4cSUAdtT"
            },
            {
              "int": "11"
            }
          ]
        }
      ]
    }
  }
}
```
is now converted to michelson as
```
{
    "parameters": "Pair \"tz1d75oB6T4zUMexzkr5WscGktZ1Nss1JrT7\" (Pair \"tz1Mj7RzPmMAqDUNFBn5t5VbXmWW4cSUAdtT\" 11)"
}
```

### PR DESCRIPTION
**Pull request by dorians**
closes #680 

Since we can't write fields as arrays or JSON I wrote `entrypoint` into the separate field.

### CODE REVIEW

File: sql/conseil.sql [hunk 1]
```diff
@@ -284,6 +284,7 @@ CREATE TABLE tezos.operations (
     amount numeric,
     destination character varying,
     parameters character varying,
+    parameters_entrypoint character varying,
```
Comment (vishakh) [inline sql/conseil.sql]: Since a transaction can apparently have multiple parameters, we should rename this field to `parameters_entrypoints`.

Comment (ivanopagano) [reply sql/conseil.sql]: I think that to add multiple entrypoints with related scripts, we also need to modify the Model that represents a transaction, for now it doesn't support an array of parameters anyway...

Comment (piotrkosecki) [reply sql/conseil.sql]: @vishakh what do you think about leaving it as `_entrypoint` instead of `_entrypoints` as @ivanopagano suggested?

Comment (vishakh) [reply sql/conseil.sql]: I think `_entrypoints` is more future proof.

Comment (piotrkosecki) [reply sql/conseil.sql]: ok, changed

**Pull request merged**